exampleSite/content/docs/development/content.md

```diff
@@ -25,6 +25,18 @@ You can change these colors by editing them in `config.toml`.
 Change other Bootstrap variables using `assets/styles/bootstrap/_variables.scss`.
 Syna customizes some parts of the theme via custom css, which is available in the `assets/styles` directory.
 
+### Fonts
+
+FontAwesome Free is supported by default and is used throughout built-in fragments. You can disable it by setting `fontawesome.disabled` in your `config.toml` to `true`.
+
+If you want to add FontAwesome Pro to the theme, the following recipe may be helpful.
+
+- Move all the files from `fontawesome-pro-[version]-web/webfonts/` (from the archive) to `static/fonts` directory in the theme.
+- Move all the files from `fontawesome-pro-[version]-web/scss/` (from the archive) to `assets/styles/fontawesome` directory in the theme.
```

Comment:
Does this have to be within the theme? Wouldn't it be enough adding it in the website so it overwrites the theme files?

Reply:
Finally had time to test and works as expected within the site. So we should definitely recommend only overwriting this within the site not the theme.

Reply:
Sorry I though `assets` directory didn't fall through as `static` directory does.